--- a/cmake/FindQtWaylandScanner.cmake
+++ b/cmake/FindQtWaylandScanner.cmake
@@ -69,7 +69,7 @@
 ecm_find_package_version_check(QtWaylandScanner)
 
 # Find qtwaylandscanner
-find_program(QtWaylandScanner_EXECUTABLE NAMES qtwaylandscanner HINTS /usr/lib64/qt5/bin/)
+find_program(QtWaylandScanner_EXECUTABLE NAMES qtwaylandscanner HINTS /usr/lib/qt5/bin/)
 
 include(FindPackageHandleStandardArgs)
 find_package_handle_standard_args(QtWaylandScanner
